configs/sst_kernel_ft-kpatch.yaml

```diff
@@ -11,8 +11,9 @@ data:
   labels:      # <-- Please leave
   - eln        #     this unchanged.
 
-#  # (Only include the following section if you have arch-specific packages.)
-#  arch_packages:
-#    x86_64:                    # <-- Architecture
-#    - arch-specific-package1   # <-- The list of binary RPM packages
-#    - arch-specific-package1   #     goes here.
+  package_placeholders:
+    # kpatch is a RHEL-only package
+    kpatch:
```

Comment:
You'll also need to remove it from the `packages` field.

You can do

```
packages: []
```

Reply:
v2: specify empty packages set as @asamalik suggested